src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -155,8 +152,7 @@ public void testArgsParsingTestnet() {
 
         iotaConfig.parseConfigFromArgs(args);
         Assert.assertEquals("port value", 14000, iotaConfig.getPort());
-        Assert.assertEquals("udp port", 13000, iotaConfig.getUdpReceiverPort());
-        Assert.assertEquals("tcp port", 27000, iotaConfig.getTcpReceiverPort());
+        Assert.assertEquals("neighboring port", 13000, iotaConfig.getNeighboringSocketPort());
         Assert.assertEquals("neighbors", Arrays.asList("udp://neighbor1", "neighbor", "tcp://neighbor2"),
```

Comment:
Are nodes specified at startup (in this case with `-n` on line 125) validated with `isURIValid` (https://github.com/iotaledger/iri/pull/1393/files#diff-e52d00ce2122bf30c0141fddce32c729R513)? This test seems to be passing since a `udp` neighbor is still being returned by `getNeighbors()` but I think it should fail since the scheme is not `tcp`.

Reply:
This particular test tests whether the command line parsing works and not whether the given values actually are valid for what they are used for (besides the type). UDP neighbors are never added to the NeighborRouter